tasks/vm.py

```diff
@@ -39,7 +39,6 @@ def create_vm(
     deploy(
         ctx,
         scenario_name,
-        key_pair_required=True,
```

Comment:
This variable is set at `True` when a scenario need a key pair so we must keep it.